.github/workflows/pr.yml

```diff
@@ -30,17 +30,16 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
-
-      - name: Launch services
-        run: |
-          AWS_ACCESS_KEY_ID=${{ secrets.AWS_ACCESS_KEY_ID }} \
-          AWS_SECRET_ACCESS_KEY=${{secrets.AWS_SECRET_ACCESS_KEY}} \
-          docker-compose up --build -d stac raster
-
       - name: Set up Python
         uses: actions/setup-python@v4
         with:
-          python-version: '3.8'
+          python-version: '3.9'
+
+      - uses: KengoTODA/actions-setup-docker-compose@v1
```

Comment:
I'm not that familiar with the subtle differences between `docker-compose` and `docker compose`. Would things work just changing the launch command to `docker compose up --build -d`? 

The docker-compose is deprecated warning on the KengoTODA page is why I ask. If it's a can of worms getting docker compose without the hyphen to work (it does for me locally anyway), let's open an issue to stop needing it in the future.

Reply:
Good catch - I had missed the warning, and also wasn't familiar with the difference of `docker-compose` vs `docker compose`